lib/cdo/properties.rb

```diff
@@ -0,0 +1,58 @@
+require 'cdo/db'
+DB = PEGASUS_DB
+
+class Properties
+  public
+
+  def initialize()
+    @table = DB[:properties]
+  end
+
+  def get(key)
+    i = @table.where(key:key.to_s).first
+    return nil unless i
+    JSON.load(StringIO.new(i[:value]))
+  end
+
+  def set(key, value)
+    key = key.to_s
+
+    i = @table.where(key:key).first
+    if i.nil?
+      @table.insert(key:key, value:value.to_json)
+    else
+      @table.where(key:key).update(value:value.to_json)
+    end
+
+    value
+  end
+
+end
+PROPERTIES = Properties.new
```

Comment:
I'd recommend just making all of these methods class methods in Properties instead of initializing another constant.